saleor/product/utils.py

```diff
@@ -72,3 +75,28 @@ def get_availability(product, discounts=None, local_currency=None):
         discount=discount,
         price_range_local_currency=price_range_local,
         discount_local_currency=discount_local_currency)
+
+
+def product_display(request, product, create=False):
```

Comment:
I'd recommend to change signature and rename `create` parameter to more verbose name, for example `create_cart`.
When you're calling `product_display(request, product, create=True)` there is no way to guess what will be created after passing `True`